### PR TITLE
Break out tabs into separate pages 

### PR DESCRIPTION
**Pull request by askiefer**
This uses @eayoungs PR and implements some of the changes that the Dept of SF requested, namely modifying the text in the side panel and replacing the circle chart with the ranking as opposed to the same information as displayed in the chart, see screenshots:

I realized I have to add the circle chart and ranking and remove the data in GHG and EUI as I did in Estar score at the top of the page...

screen shot 2017-07-10 at 4 48 26 pm

screen shot 2017-07-10 at 4 47 22 pm
screen shot 2017-07-10 at 4 47 15 pm

I'm going to make a few issues on the remaining to-dos @eayoungs @thfield 

cc @smoningi

### CODE REVIEW

File: dashboard/estar.html [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+      	<div class="col-xs-3">
+			<div class="row">
+	            <div class="panel-body side flex-grow">
+                	<div class="col-xs-12">
+                		<div class="row center">
+                			<div class="profile-header">BENCHMARKING PROFILE</div>
+                		</div>
+                		<div class="row center">
+                			<i class="fa fa-building" aria-hidden="true"></i>
+                			<!-- <img src="../assets/hotel-icon.svg" alt="hotel-icon" id="building-icon"> -->
```
Comment (thfield) [inline dashboard/estar.html:81]: is the font-awesome image the new path forward?  it's only changed on this page.

**Review by thfield — APPROVED** [review]
lgtm!

i think @eayoungs mentioned splitting the sidebar html out into its own file; i was thinking the other day about splitting up `dashboard.js` into separate modules and using webpack, i'll open an issue for this

**Comment by eayoungs** [comment]
Did you decide with Ammon & Barry whether to have the 'bar' over the 'wedge'? Everything looks good to me but I seem to have lost track of the details of the presentation.

**Review by eayoungs — APPROVED** [review]
Looks good! Just FYI: @thfield mentioned that he expected you, @askiefer, to close the PR, once approved.

**Comment by thfield** [comment]
one of my many recent PRs changes the bar into a wedge, FYI

**Comment by askiefer** [comment]
Awesome thanks @eayoungs @thfield closing now!

File: dashboard/estar.html (hunk 1 above)
Comment (askiefer) [reply dashboard/estar.html:81]: ah yes thank you I'll change it on the others and submit a new PR

**Pull request merged**